### ISSUE TITLE
Write a Introduction document

### ISSUE THREAD

**Issue by adler-j** [odlgroup/odl#5]
We need something that users can look at to see where to get started, similar to [numpys quickstart](https://docs.scipy.org/doc/numpy-dev/user/quickstart.html)

Lets make a short list of what this should contain (obviously inspired by numpy)
- A short note on pre-reqs, mainly some programming knowledge and a note that we try to follow numpys conventions where possible, also some words on the math needed and some links.
- Introduce a `LinearSpace`, quite obviously `Rn`, show that you can do what you'd expect with it.
  - Space creation
  - Vector creation, `element`, `zero`.
  - Workhorse methods (`+`, `-`, `/`, `*`), indexing
  - Core methods `lincomb`, `multiply`.
  - Show how to interact with `numpy`. `asarray`, `__array_wrap__` to use `ufunc`'s
  - printing arrays
- Introduce a `Set`, I'd prefer `Interval(0, 1)`, do some simple things with it.
  - `__contains__`
- Define `L2(Interval(0, 1))`
  - create continuous function
  - show function arithmetic
- Discretize `l2_uniform_discretization(L2(Interval(0, 1)))`
  - Show that norms are mapped as expected etc
  - Show `Rn`/`CudaRn` possibility
  - Show complex
  - Show the `odl.diagnostics` possibilities.
- Define a `Operator` on the space
  - Introduce `Operator`
  - Define a very simple operator, perhaps the convolution
  - Show some operator magic (perhaps addition, or scaling)
- Solve an inverse problem
  - I realy like the deconvolution problem here.

**Comment by adler-j** [comment]
This is started in the files

[about odl](https://github.com/odlgroup/odl/blob/master/doc/source/guide/introduction/about_odl.rst)

where we introduce the concepts in very simple ways, and in

[getting started](https://github.com/odlgroup/odl/blob/master/doc/source/guide/introduction/getting_started.rst)

where we show the users how to get started solving actual problems.

**Comment by adler-j** [comment]
This has to be done before the article is published.

**Comment by adler-j** [comment]
Well isn't this kinda done, or what is needed here?

**Comment by kohr-h** [comment]
I think we have covered all those parts, although I feel they could be better organized. But that would be a topic for a new issue.
